Formula/tctl.rb

```diff
@@ -0,0 +1,27 @@
+class Tctl < Formula
+  desc "Temporal CLI (tctl)"
+  homepage "https://temporal.io/"
+  url "https://github.com/temporalio/temporal/archive/v1.5.1.tar.gz"
+  sha256 "a2f75873d7c211e07c40653c605e64fcf732543c9d50683b2d61a985f6b3f37f"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
```

Comment:
```suggestion
    ENV["GOPATH"] = buildpath
    ENV["GO111MODULE"] = "auto"
```
I think you need this to compile on ARM with Go 1.16. However, let's wait and see if it's needed first.

Reply:
Ok, doesn't look like this one is needed actually.

Reply:
Why do you need GOPATH as you already have the go mod builds?